### ISSUE TITLE
[V3 Fuzzy help] Option to disable

### ISSUE THREAD

**Issue by laggron42** [Cog-Creators/Red-DiscordBot#1736]
# Feature request

#### Select the type of feature you are requesting:

- [ ] Cog
- [ ] Command
- [ ] API functionality

#### Describe your requested feature

The fuzzy search was recently added. While it is actually a cool feature, there should be a way to disable it. It is pretty annoying for me on my server (like with random members who spam it).

If I say `...` in the chat (my prefix is `.`), the bot will send the fuzzy help. If I use a custom command, the bot also send the fuzzy help. Ok that one is a bug, but I still find it annoying on some servers, and we should be able to disable it, instead of editing and commenting a line...

**Comment by laggron42** [comment]
Closing this since it is removed now.

**Comment by Tobotimus** [comment]
@retke We keep issues updated in parallel with what's on V3/develop (or develop for V2). On V3/develop, fuzzy search is implemented but not toggleable. The toggle is included in #1788, however this issue shouldn't be closed until that is merged

**Comment by laggron42** [comment]
I though the fuzzy help was going to be removed. I guess there's another issue to open now.

**Comment by palmtree5** [comment]
nah, a release was done without fuzzy because of the problems it was causing (which wound up causing other issues, but that's a different story). It was never removed from V3/develop though as the intention was to fix it up for beta 16

**Comment by palmtree5** [comment]
At any rate, this has been implemented in #1788
